docs/logging.md

```diff
@@ -1,58 +1,68 @@
 # Logging
 
-Using Bigflow, you can easily add a handler to your root logger that will send all logging calls to the Cloud Logging.
-In the CL, you can filter logs per project id/logger name and workflow id. 
 
-## Getting started
-To use logging, you have to install `bigflow` with `logger` module - `bigflow[log]`.
-To create a logger with the CL handler, you have to use `configure_logging` method.
+With Bigflow you can easily configure Python logging to send all messages the Cloud Logging (Stackdriver).
+Cloud Logging allows you to filter logs by project id / log name / workflow id and other fields.
 
-```python
-from bigflow.log import BigflowLogging
-project_id = 'some_project'
-logger_name = __name__
-
-
-BigflowLogging.configure_logging(project_id, logger_name)
-```
+To use logging, you have to install `bigflow` with `log` module - `bigflow[log]`.
```

Comment:
To use logging, you have to install `bigflow` with `log` module:

`pip install bigflow[log]`